core/workspace_svg.js

```diff
@@ -1726,7 +1726,11 @@ Blockly.WorkspaceSvg.prototype.getBlocksBoundingBox = function() {
 
   // Start at 1 since the 0th block was used for initialization.
   for (var i = 1; i < topElements.length; i++) {
-    var blockBoundary = topElements[i].getBoundingRectangle();
+    var topElement = topElements[i];
+    if (topElement.isInsertionMarker && topElement.isInsertionMarker()) {
```

Comment:
Choosing to check here, rather than not allowing insertion markers to be added to top bounded elements, in order to avoid any performance implications of needing to remove the block from top bounded elements whenever an insertion marker is created.
An insertion marker is created as a regular block (would be added to top elements) and then set as an insertion maker (where we would need to remove it from top elements). Removing from top elements would require traversing through all the top elements (with the `remove` call).